src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/visualizers/tools/measure-distance.visualizer.ts

```diff
@@ -9,52 +9,46 @@ import Circle from 'ol/style/Circle';
 import Point from 'ol/geom/Point';
```

Comment:
The file compare says that this file especially changed a lot. What is the reason for this?

Reply:
there was problem with the OL v.6 some we change how the measure work.
maybe this led to many change in this file.

Reply:
Do you know what specifically was the problem with OL6 here, and what we changed in how the measure works?

Reply:
@tltk90

Reply:
yes. we remove some interaction while ol iterate over the interaction array,
and it make some error for ol try call to function from undefined object.

so i change that all the measure act with one interaction instead that every measure create it's own interaction